test/templates/side-nav/index.njk

```diff
@@ -7,42 +7,196 @@
 {% set example %}
 <nav class="fd-side-nav">
     <div class="fd-side-nav__main-navigation">
-      <ul class="test-sidenav-list">
-        <li>Item 1</li>
-        <li>Item 1</li>
-        <li>Item 1</li>
-        <li>Item 1</li>
-      </ul>
```

Comment:
We shouldn't bring the nested list in sidenav. There are tests for Nested list. Inside Sidenav tests we should only test the Sidenav styling because our components are self-contained. You should revert the test changes

Reply:
Restored previous test cases